rds/rds.tf

```
@@ -6,7 +6,7 @@ resource "aws_db_instance" "main" {
   engine_version                      = var.is_read_replica ? null : var.engine_version
   instance_class                      = var.instance_class
   identifier                          = "${var.project}-${var.environment}${var.is_read_replica ? "-read-replica" : ""}"
-  db_name                             = var.is_read_replica ? null : "${var.project}_${var.environment}" # name of the initial database
+  db_name                             = var.is_read_replica ? null : replace("${var.project}_${var.environment}", "/[^0-9A-Za-z_]/", "_") # name of the initial database
```

```suggestion
  db_name                             = var.is_read_replica ? null : replace("${var.project}_${var.environment}", "/[^0-9A-Za-z]/", "_") # name of the initial database
```

replacing `_` with `_` is redundant